src/xrCore/XML/tinyxmlparser.cpp

```diff
@@ -100,22 +100,22 @@ void TiXmlBase::ConvertUTF32ToUTF8(unsigned long input, char* output, int* lengt
 
     output += *length;
 
-    // Scary scary fall throughs.
+    // Scary scary fallthroughs.
```

Comment:
Changing it here is not needed :)
```suggestion
    // Scary scary fall throughs.
```